### PR TITLE
fix(breadcrumb): change template to nav

### PR DESCRIPTION
**Pull request by YonatanKra**

### CODE REVIEW

File: libs/components/src/lib/breadcrumb-item/breadcrumb-item.spec.ts [hunk 1]
```diff
@@ -189,4 +191,20 @@ describe('vwc-breadcrumb-item', () => {
 			expect(anchorElement?.getAttribute(attribute)).toEqual(text);
 		});
 	});
+
+	describe('a11y', () => {
+		it('should pass accessibility test', async () => {
+			const { shadowRoot } = element;
+			if (!shadowRoot) { return; }
+
+			const results = await axe(shadowRoot.innerHTML, {
```
Comment (yinonov) [inline libs/components/src/lib/breadcrumb-item/breadcrumb-item.spec.ts:200]: Can we check breadcrumb with its items?
Need to check how's it testing the items templates as well

File: libs/components/src/lib/breadcrumb/breadcrumb.template.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+/**
+ * The template for the {@link @microsoft/fast-foundation#Breadcrumb} component.
+ *
+ * @param context
+ * @param definition
+ * @public
+ */
+export const breadcrumbTemplate: FoundationElementTemplate<ViewTemplate<Breadcrumb>> = () => html`
+    <nav role="navigation" class="base">
+        <div role="list" class="list" part="list">
```
[the remark this answers is not in the document]
Comment (YonatanKra) [reply libs/components/src/lib/breadcrumb/breadcrumb.template.ts:15]: Yes - this is how `fast` implemented it.
I think UL might be more fitting, but don't we go with `fast` on such decisions?

File: libs/components/src/lib/breadcrumb-item/breadcrumb-item.spec.ts (hunk 1 above)
Comment (YonatanKra) [reply libs/components/src/lib/breadcrumb-item/breadcrumb-item.spec.ts:200]: It is tested with its children:
```
<vwc-breadcrumb>
 <vwc-breadcrumb-item href="#" text="breadcrumb"></vwc-breadcrumb-item>
    <vwc-breadcrumb-item text="..."></vwc-breadcrumb-item>
    <vwc-breadcrumb-item href="#" text="breadcrumb"></vwc-breadcrumb-item>
    <vwc-breadcrumb-item text="breadcrumb"></vwc-breadcrumb-item>
<vwc-breadcrumb>
```

File: libs/components/src/lib/breadcrumb-item/breadcrumb-item.spec.ts [hunk 3]
```diff
@@ -189,4 +190,23 @@ describe('vwc-breadcrumb-item', () => {
 			expect(anchorElement?.getAttribute(attribute)).toEqual(text);
 		});
 	});
+
+	describe('a11y', () => {
+		it('should pass accessibility test', async () => {
```
Comment (yinonov) [inline libs/components/src/lib/breadcrumb-item/breadcrumb-item.spec.ts:195]: this should actually fail cause list items with aren't nested in list are wrong

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/breadcrumb-item/breadcrumb-item.spec.ts (hunk 3 above)
Comment (YonatanKra) [reply libs/components/src/lib/breadcrumb-item/breadcrumb-item.spec.ts:195]: It doesn't fail though... an issue with `axe`?

**Review by yinonov — APPROVED** [review]

**Pull request merged**